CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ This change log adheres to standards from [Keep a CHANGELOG](http://keepachangel
 - [`no-namespace`]: Make rule fixable ([#1401], thanks [@TrevorBurnham])
 - support `parseForESLint` from custom parser ([#1435], thanks [@JounQin])
 - [`no-extraneous-dependencies`]: Implement support for [bundledDependencies](https://npm.github.io/using-pkgs-docs/package-json/types/bundleddependencies.html) ([#1436], thanks [@schmidsi]))
+- [`order`]: Adds support for pathGroups to allow ordering by defined patterns ([#795], thanks [@Mairu]
```

Comment:
```suggestion
- [`order`]: Adds support for pathGroups to allow ordering by defined patterns ([#795], thanks [@Mairu])
```

You'll need to add links for 795, 1386, and for @Mairu, in this file.

Reply:
Added them, though that was some kind of magic :)